oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -254,6 +254,15 @@ pulsar-consumer:
 Ehcache:
   id: 75
   languages: Java
+servicecomb-mesher:
+  id: 76
+  languages: Golang
+servicecomb-service-center:
+  id: 77
+  languages: Golang
```

Comment:
Golang should take ` [5000, 6000)`, such as node js
```
# NoeJS components
# [4000, 5000) for Node.js agent
```

Reply:
> Golang should take ` [5000, 6000)`, such as node js
> 
> ```
> # NoeJS components
> # [4000, 5000) for Node.js agent
> ```

When I add componetId in https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml#L113.

Add icon in https://github.com/apache/skywalking-ui/tree/master/public/img/node .

Could you teach me how to establish the relationship between icon in public/img/node and component ID？

I want to show this icon in topology when I send corresponding component ID.